src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -375,6 +375,11 @@ public bool TryGetFunctionWorkspace(Guid id, bool isTestMode, out CustomNodeWork
             return false;
         }
 
+        public bool TryGetFunctionWorkspace(Guid id, bool isTestMode, out ICustomNodeWorkspaceModel ws)
+        {
+            return TryGetFunctionWorkspace(id, isTestMode, out ws);
+        }
```

Comment:
That's exactly what I've done. Now in `ICustomNodeManager` there is only one such method:

```c#
bool TryGetFunctionWorkspace(Guid id, bool isTestMode, out ICustomNodeWorkspaceModel ws);
```